packages/devextreme/js/__internal/grids/grid_core/editing/m_editing_form_based.ts

```diff
@@ -563,6 +563,18 @@ export const editingFormBasedModule = {
             this.callBase($cell, parameters);
           }
         },
+        _updateContent() {
+          const editingController = this._editingController;
```

Comment:
minor: we usually (in new code) avoid creating variables like this and just use `this._editingController`: "editingController" isn't much shorter, but loses context (where we got this variable from) 
```suggestion
          const editingController = this._editingController;
```